pavelib/assets.py

```diff
@@ -447,6 +449,10 @@ def update_assets(args):
         '--watch', action='store_true', default=False,
         help="Watch files for changes",
     )
+    parser.add_argument(
```

Comment:
@benpatterson Yes, makes sense to me that you can choose to do both. I think it would also be okay that if you specify a log file then that means it won't also pipe to the console. That seems like an explicit request that you want it logged to a file. So whichever way you choose is fine with me.